app/src/main/java/mozilla/lockbox/action/RouteAction.kt

```diff
@@ -31,6 +32,12 @@ open class RouteAction(
     data class DisplayItem(val id: String) : RouteAction(TelemetryEventMethod.show, TelemetryEventObject.entry_detail)
     data class EditItem(val id: String) : RouteAction(TelemetryEventMethod.show, TelemetryEventObject.edit_entry_detail)
 
+    data class ShowToastNotification(
+        @StringRes val strId: Int? = null,
+        val text: String? = null,
+        val viewGroup: ViewGroup
```

Comment:
Hmm.

Passing around a `ViewGroup` like this looks… weird.

I'd recommend a `sealed class` in the same way as we do for `Dialog`s: so all the `strId`s are defined in the same place.